site/content/docs/multistep-checks/example-checks.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Example checks
+weight: 22
+menu:
+  resources:
+    parent: "Multistep checks"
+    identifier: "multistep-example-checks"
+cli: true
+---
+
+Below are a number of checks showcasing how to use Multistep checks. These are similar to the templates provided in Checkly when creating a new Multistep check and should give you an idea of how to get started building various types of API monitors.
```

Comment:
@drakirnosslin nit: I would stick to "API checks" instead of "API monitors" for consistency. A less experiences user might get confused thinking these are two separate things.